### PR TITLE
Add weekly update 2016-03-01

### PR DESCRIPTION
**Pull request by JungMinu**
Please review this post. :heart:,@nodejs/evangelism

### CODE REVIEW

**Comment by ghost** [comment]
[Travis build passed](https://travis-ci.org/nodejs/nodejs.org/builds/112859301) :+1:

**Comment by JungMinu** [comment]
@hg Thanks, fixed :smile:

**Comment by ghost** [comment]
LGTM otherwise!

File: locale/en/blog/weekly-updates/weekly-update.2016-03-01.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+category: weekly
+slug: weekly-update-2016-03-01
+layout: blog-post.hbs
+---
+
+### Node.js News
+
+### OpenSSL updates, 1.0.2g and 1.0.1s
+
+Our crypto team will make an impact assessment for Node.js users of the OpenSSL releases. This information may vary depending for the different active release lines and will be posted here.
```
Comment (thefourtheye) [inline locale/en/blog/weekly-updates/weekly-update.2016-03-01.md]: depending on the?

**Comment by JungMinu** [comment]
Thanks, updated :)

**Pull request merged**